DSA/3.2 Greedy Algorithms/22. Graph Coloring/3.2.22_Graph_Coloring_Problem.md

```diff
@@ -0,0 +1,5 @@
+3.2.22_Grapgh_Coloring_Problem.md
```

Comment:
Add some description about the content of file and mention the problem statement. Add link like this
[Audio File](https://youtu.be/Pb0ZneYKAYI)